File: src/main/java/ApplicationsPage.java

import com.codeborne.selenide.SelenideElement;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;

import static com.codeborne.selenide.Selectors.byXpath;
import static com.codeborne.selenide.Selenide.$;
import static org.testng.Assert.assertTrue;
public class ApplicationsPage extends BasePage{
    public By tenderLink = By.xpath("//li[@id='docia_tender']");
    private final static String TITLE = "Applications page";
    public ApplicationsPage(WebDriver driver) {
        super(driver);
    }
    public WebElement tenderLink()
    {
        return driver.findElement((By) tenderLink);
    }
}

File: src/main/java/main/ApplicationsPage.kt
package main

import BasePage
import org.openqa.selenium.By
import org.openqa.selenium.WebDriver
import org.openqa.selenium.WebElement

class ApplicationsPage(driver: WebDriver?) : BasePage(driver) {
    var tenderLink: By = By.xpath("//li[@id='docia_tender']")
    fun tenderLink(): WebElement {
        return driver!!.findElement(tenderLink)
    }

    companion object {
        private const val TITLE = "Applications page"
    }
}
